lintly/parsers.py

```diff
@@ -234,6 +234,57 @@ def parse_violations(self, output):
         return violations
 
 
+class BanditJSONParser(BaseLintParser):
+    """
+    Bandit JSON format:
+
+      [
+          {{
+      "code": "13 \n14 env = Environment(\n15     loader=FileSystemLoader(TEMPLATES_PATH),\n16     autoescape=False\n17 )\n",
+      "filename": "./lintly/formatters.py",
+      "issue_confidence": "HIGH",
+      "issue_severity": "HIGH",
+      "issue_text": "Using jinja2 templates with autoescape=False is dangerous and can lead to XSS. Use autoescape=True or use the select_autoescape function to mitigate XSS vulnerabilities.",
```

Comment:
Lines that are very long makes the code more annoying to read. Most people like to keep their line length at aroudn 80-100 characters.

Since this line isn't *that* important for understanding the format, truncating the line (and ending it with something like "...") will get the point across, e.g.,
`"issue_text": "Using jinja2 templates with autoescape=False is dangerous ...",`